src/test/README.md

```diff
@@ -100,3 +131,7 @@ gdb src/test/test_bitcoin core
 
 (gbd) bt  # produce a backtrace for where a segfault occurred
 ```
+
+#### Additional guide
+Below is a great guide on how to run the tests if you're not able to compile the tests successfully yet:
+https://jonatack.github.io/articles/how-to-compile-bitcoin-core-and-run-the-tests
```

Comment:
NACK on adding links to personal sites, blogs etc.

Although, https://jonatack.github.io/articles/how-to-compile-bitcoin-core-and-run-the-tests indeed is a great piece to read for newbies.